bin/test-documentation

```diff
@@ -5,4 +5,4 @@
 runSbt unidoc
 
 cd docs
-runSbt markdownValidateDocs test markdownEvaluateSbtFiles
+runSbt -Dakka.test.single-expect-default=15s markdownValidateDocs test markdownEvaluateSbtFiles
```

Comment:
👍🏼

Reply:
Doesn't seem to have worked 😦 https://travis-ci.org/lagom/lagom/jobs/355682098

Reply:
Because tests are forked 🤦‍♂️